src/constraints.jl

```diff
@@ -172,6 +172,22 @@ function delete(model::Model, con_ref::ConstraintRef{Model})
     MOI.delete(backend(model), index(con_ref))
 end
 
+"""
+    delete(model::Model, con_refs::Vector{ConstraintRef})
+
+Delete the constraints associated with `con_refs` from the model `model`.
+Solvers may implement methods for deleting multiple constraints that are
+more efficient than repeatedly calling the single constraint delete method.
+"""
+function delete(model::Model, con_refs::Vector{<:ConstraintRef{Model}})
+    # This is just a fallback of the fallback. To be really useful,
+    # instead of calling JuMP.delete repeatedly it needs to call a
+    # (currently unimplemented) fallback in MOI like the one for
+    # batch variable deletion: https://github.com/JuliaOpt/MathOptInterface.jl/blob/78c15baee/src/indextypes.jl#L119
+    # Then the solvers may reimplement the fallback in MOI level.
+    delete.(model, con_refs)
```

Comment:
https://github.com/JuliaOpt/MathOptInterface.jl/blob/2a8603407fdabce4f322cd90fe646f3a0efdc6a6/src/indextypes.jl#L126-L130
The MOI fallback actually covers constraint indices.

You you need something like:
```Julia
function delete(model::Model, con_refs::Vector{<:ConstraintRef})
    if any(c -> model !== c.model, con_refs)
        error("A constraint reference you are trying to delete does not belong to the model.")
    end
    MOI.delete(backend(model), index.(con_refs))
    return
end
```

Reply:
Will do. In the meanwhile, can you check the doubts from my last comment?

Reply:
Done.